app/src/main/java/edu/ucsb/cs56/projects/android/conversion_showdown/MyUtils.java

```diff
@@ -17,19 +17,11 @@
 public class MyUtils {
 
     //The characters shown are basically what are allowed
-    final public static ArrayList<Character> symbols = new ArrayList<>( Arrays.asList(new Character[]{
-            '0', '1', '2', '3' , '4' , '5', '6' , '7' , '8' , '9',
-            'A', 'B', 'C', 'D', 'E', 'F'
-    }));
-
-    static TreeSet<Integer> baseWhiteList = new TreeSet<Integer>(){
-        {
-            add( 2 );
-            add( 8 );
-            add( 10 );
-            add( 16 );
-        }
-    };
+    final public static ArrayList<Character> symbols = new ArrayList<>(Arrays.asList(new Character[]{
+                '-', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
+                'A', 'B', 'C', 'D', 'E', 'F'
```

Comment:
There should be support for small letter values for hex number input.

Reply:
It has already supported lower letters. The line 81 demonstrates how.

Reply:
Okay, line 81 supports lower letters, but not upper letters as it converts to lower case first and then matches. But, considering it for this review.